BasicSample/versions.gradle

```diff
@@ -136,6 +137,7 @@ deps.dexmaker = "com.linkedin.dexmaker:dexmaker-mockito:$versions.dexmaker"
 deps.constraint_layout = "androidx.constraintlayout:constraintlayout:$versions.constraint_layout"
 deps.timber = "com.jakewharton.timber:timber:$versions.timber"
 deps.junit = "junit:junit:$versions.junit"
+deps.junit_androidx = "androidx.test.ext:junit:$versions.junit_androidx"
```

Comment:
this should be in the atsl block above, and use the same version

Reply:
:+1: